src/resources/Interfaces.ts

```diff
@@ -167,8 +167,15 @@ export interface InitialState {
   session: {
     isLoggedIn: boolean
     userInfo: UserInfo
-    lightningPayEnabled: boolean
-    boostAmount: number
+    valueSettings: {
```

Comment:
ValueSettings sounds a little too generic here. Maybe rename it to `ValueTagSettings` or something more specific?